static/app/views/setupWizard/wizardProjectSelection.tsx

```diff
@@ -277,141 +278,139 @@ export function WizardProjectSelection({
+                children: isCreateProjectSelected
+                  ? t('Create Project')
+                  : selectedProject?.slug || (
+                      <SelectPlaceholder>{t('Select a project')}</SelectPlaceholder>
+                    ),
+              }}
+              onChange={({value}) => {
+                setSelectedProjectId(value as string);
+              }}
+              emptyMessage={emptyMessage}
+              menuFooter={
+                isCreationEnabled
+                  ? ({closeOverlay}) => (
+                      <Flex justify="end">
+                        <Button
+                          size="xs"
+                          onClick={() => {
+                            setSelectedProjectId(CREATE_PROJECT_VALUE);
+                            closeOverlay();
+                          }}
+                          icon={<IconAdd isCircled />}
+                        >
+                          {t('Create Project')}
+                        </Button>
+                      </Flex>
+                    )
+                  : undefined
+              }
+            />
+          )}
+        </FieldWrapper>
+        {isCreateProjectSelected &&
+          (isOrgMemberWithNoAccess ? (
+            projectNameField
+          ) : (
+            <Columns>
+              {projectNameField}
+              <FieldWrapper>
+                <label>{t('Team')}</label>
+                <StyledCompactSelect
+                  value={newProjectTeam as string}
+                  options={
+                    selectableTeams?.map(team => ({
+                      value: team.slug,
+                      label: `#${team.slug}`,
+                      leadingItems: <IdBadge team={team} hideName />,
+                      searchKey: team.slug,
+                    })) || []
+                  }
+                  triggerProps={{
+                    icon: selectedTeam ? (
+                      <IdBadge avatarSize={16} team={selectedTeam} hideName />
+                    ) : null,
+                    children: selectedTeam ? `#${selectedTeam.slug}` : t('Select a team'),
+                  }}
+                  onChange={({value}) => {
+                    setNewProjectTeam(value as string);
+                  }}
+                />
+              </FieldWrapper>
+            </Columns>
+          ))}
+        <SubmitButton
+          disabled={!isFormValid || isPending}
+          priority="primary"
+          type="submit"
+        >
+          {t('Continue')}
+        </SubmitButton>
+      </Stack>
+    </form>
   );
 }
 
-const StyledForm = styled('form')`
-  display: flex;
-  flex-direction: column;
-  gap: ${space(2)};
-`;
-
 const Heading = styled('h5')`
   margin-bottom: ${space(0.5)};
 `;
 
-const FieldWrapper = styled('div')`
-  display: flex;
-  flex-direction: column;
-  gap: ${space(0.5)};
-`;
+function FieldWrapper(props: React.ComponentProps<typeof Stack>) {
+  return <Stack gap="xs" {...props} />;
+}
```

Comment:
Inline

Reply:
There are quite a few instances of this component usage on this page, so I'd rather keep this as is to make it less prone to updates